src/test/java/com/moodybluez/enterprise/EnterpriseApplicationTests.java

```diff
@@ -43,14 +41,17 @@ void fetchMoodByID_returnsSadForID3() {
 	}
 
 	private void givenMoodDataAreAvailable() {
+		mood.setMood("Sad");
+		mood.setMoodID(3);
+		moodDAO.createEntry(mood);
```

Comment:
This method was empty. You need to use this to initialize your objects. As the method states - It is a given Data.